apps/web/src/components/media-gallery/media-gallery.tsx

```diff
@@ -102,10 +102,10 @@ export default function MediaGallery({ media }: MediaGalleryProps) {
                 onClick={() => handleMediaChange(media)}
               >
                 <Image
-                  alt=""
+                  alt="media"
```

Comment:
Let's add a translation for the `alt` attribute, something like this maybe?

```suggestion
                  alt={t('common:actions.View image X of Y')}
```